superset-frontend/src/dashboard/components/nativeFilters/FilterBar.tsx

```diff
@@ -399,6 +399,16 @@ const FilterBar: React.FC<FiltersBarProps> = ({
     });
   };
 
+  const handleResetAll = () => {
+    setFilterData({});
+    const filterIds = Object.keys(filterData);
+    filterIds.forEach(filterId => {
+      if (filterData[filterId]) {
+        setExtraFormData(filterId, {});
```

Comment:
Not really relevant just yet as we don't have default values implemented, but I wonder if "reset all" should reset to defaults or remove all selected values?

Reply:
hmm indeed - I assumed that reset meant resetting everything to 0, but maybe the user really expects a reset to default? Adam raised this issue in all PR with native filters. 
Thanks for the merging, I'll improve it later if it the assumptions are different.

Reply:
Yeah, I think we need to discuss this more later.